mongock-core/mongock-runner/mongock-runner-core/src/main/java/io/mongock/runner/core/executor/system/changes/SystemChangeUnit10001.java

```diff
@@ -0,0 +1,41 @@
+package io.mongock.runner.core.executor.system.changes;
+
+import io.mongock.api.annotations.BeforeExecution;
+import io.mongock.api.annotations.ChangeUnit;
+import io.mongock.api.annotations.Execution;
+import io.mongock.api.annotations.RollbackBeforeExecution;
+import io.mongock.api.annotations.RollbackExecution;
+import io.mongock.driver.api.common.SystemChange;
+import io.mongock.driver.api.entry.ChangeEntryService;
+
```

Comment:
Add comment before the class, explaining what this changeUnit does. SOmething like:
/**
THis ChangeUnit ensures the systemChange field is present in the table changeLogItems and set the default values: True tu all systemChangeUnits and false to all those are user nomal

TODO: whatever
**/